cosmwasm/ibc-union/lightclient/berachain/src/client.rs

```diff
@@ -5,7 +5,7 @@ use ethereum_light_client_types::StorageProof;
 use ibc_union_light_client::{
     spec::Timestamp, ClientCreationResult, IbcClient, IbcClientCtx, IbcClientError, StateUpdate,
 };
-use ibc_union_msg::lightclient::Status;
+use ibc_union_spec::path::Status;
```

Comment:
no need to add this as a dependency, `ibc_union_light_client` reexports `ibc_union_spec` under `ibc_union_light_client::spec` (as you can see in the import above this one)